includes/wc-internal/trunkrs-order.php

```diff
@@ -54,7 +54,8 @@ private function init($withMeta)
 
       foreach ($shippingItem as $item) {
         $shippingMethodId = $item->get_method_id();
-        if ($shippingMethodId !== TRUNKRS_WC_Bootstrapper::DOMAIN) {
+        if ($shippingMethodId !== TRUNKRS_WC_Bootstrapper::DOMAIN
```

Comment:
Also goes for this one!

Reply:
`$shippingMethodId !== TRUNKRS_WC_Bootstrapper::DOMAIN` my understanding about this is that when shipping method is not equal to trunkrs we will skip the processing of order for shipping.